IntelliTect.TestTools.Selenate/ElementsHandler.cs

```diff
@@ -104,32 +104,41 @@ public bool ContainsText(string text)
         }
 
         /// <summary>
-        /// Returns an ElementHandler as long as a matching DOM item is found, based on XPath or CSS index.
+        /// Returns an ElementHandler as long as a matching DOM item is found, based on XPath or CSS index. <br />
+        /// Note that if your locator does not have '{index}' in it, one will be appended to the end. <br />
+        /// E.G. The locator By.CssSelector("div>div") will become By.CssSelector("div>div:nth-of-type(1)") <br />
+        /// and By.CssSelector("div{index}>div") will become By.CssSelector("div:nth-of-type(1)>div") <br /> <br />
         /// Please ensure at least two elements can be found when attempting to use this method.  <br />
-        /// Respects any timeout set for this ElementsHandler. <br />
+        /// Respects any timeout set for this ElementsHandler. <br /> <br />
         /// NOTE: By.Name locators have not yet been verified to work. Please file an issue if one is encountered: https://github.com/IntelliTect/TestTools.Selenate/issues
         /// </summary>
         /// <returns>The enumerable of ElementHandlers that exist at the time of invocation.</returns>
-        public IEnumerable<ElementHandler> GetElementHandlers()
+        public IEnumerable<ElementHandler> GetElementHandlers(By? byOverride = null)
```

Comment:
Should we add xml docs (`<param name ="byOverride">...</param>`) for this parameter too?